include/boost/geometry/strategies/geographic/distance.hpp

```diff
@@ -22,7 +22,7 @@
 #include <boost/geometry/core/radius.hpp>
 
 #include <boost/geometry/formulas/andoyer_inverse.hpp>
-#include <boost/geometry/formulas/elliptic_arc_length.hpp>
+#include <boost/geometry/formulas/elliptic_meridian_arc_inverse.hpp>
```

Comment:
Btw, for consistency, shouldn't it be called "spheroidal meridian arc"? Are we ok with using "spheroid" and "ellipsoid" interchangeably? It would probably only be a problem if we wanted to support 3-axial ellipsoids.

Reply:
ok for now I simply use the meridian_direct and meridian_inverse